src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int RESIZE_INDEX = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshHold;
```

Comment:
Typo mistake, also this is one word, not 2
```suggestion
    private int threshold;
```